File: prueba/Cliente/Web/IntegradorRecaudo/src/app/data/schema/versionesDto.ts
export class VersionesDto {
  id?: number;
  fecha?: string;
  nombre_version?: string;
  java_version?: string;
  php_version?: string;
  movil_version?: string;
  activo?: boolean;

  constructor(init?: Partial<VersionesDto>) {
    if (init) {
      Object.assign(this, init);
    }
  }
}

File: prueba/Servidor/Core/integradorrecaudo/src/main/java/integradorrecaudo/versiones/dto/VersionesDTO.java
package integradorrecaudo.versiones.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

public record VersionesDTO(

        Integer id,
        @JsonProperty("nombre_version")
        String nombreVersion,
        @JsonProperty("java_version")
        String javaVersion,
        @JsonProperty("php_version")
        String phpVersion,
        @JsonProperty("movil_version")
        String movilVersion,
        String version,
        String fecha,
        boolean activo
) {


}
